shared/common-adapters/zoomable-box.android.tsx

```diff
@@ -66,53 +75,40 @@ export function ZoomableBox(props: Props) {
 
     // we only activate pan handler when the image is zoomed or user is not pinching
     const pan = Gesture.Pan()
-      .onStart(() => {
-        if (isPinching.value || !isZoomed.value) return
-
+      .averageTouches(true)
+      .onStart(e => {
+        if (!isZoomed.value) {
+          return
+        }
         cancelAnimation(translationX)
         cancelAnimation(translationY)
         cancelAnimation(scale)
 
         prevTranslationX.value = translationX.value
         prevTranslationY.value = translationY.value
+        panFingers.value = e.numberOfPointers
       })
       .onUpdate(e => {
-        if (isPinching.value || !isZoomed.value) {
+        // if we're done panning ignore us letting go
+        if (e.numberOfPointers < panFingers.value) {
```

Comment:
this is so when you transition from a pinch to just fingers we don't pan around after, its too jarring